torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1260,6 +1260,28 @@ def reference_sigmoid(x):
             return (1 / (1 + np.exp(-x)))
         return scipy.special.expit(x)
 
+    def reference_lgamma(x):
+        # scipy.special.gammaln returns `-inf` when input is `-inf`.
```

Comment:
This is a bug in SciPy.

scipy.special.gammaln is documented as equivalent to math.lgamma, but math.lgamma returns inf when given -inf. 

Would you file an issue with SciPy, @kshitij12345?

cc @rgommers